src/util/std_expr.h

```diff
@@ -1920,6 +2182,16 @@ inline and_exprt &to_and_expr(exprt &expr)
   return static_cast<and_exprt &>(expr);
 }
 
+template<> inline bool check_expr_type<and_exprt>(const exprt &base)
+{
+  return base.id()==ID_and;
+}
+// inline void validate_expr(const and_exprt &value)
```

Comment:
Delete or uncomment (other commented validate operations too)

Reply:
As discussed, these match the commented code above.